opengrok-indexer/src/main/java/org/opengrok/indexer/web/messages/Message.java

```diff
@@ -183,11 +193,46 @@ public void serialize(
         public Duration deserialize(final JsonParser parser, final DeserializationContext context)
                 throws IOException {
             try {
+                // TODO Util.decode()
                 return Duration.parse(context.readValue(parser, String.class));
             } catch (DateTimeParseException e) {
                 throw new IOException(e);
             }
         }
     }
 
+    private static class HTMLSerializer extends StdSerializer<String> {
```

Comment:
why we need a html serializer? why is necessary to return encoded html markup in the message?

Reply:
this used to be there originally - see http://demo.opengrok.org/xref/opengrok/opengrok-indexer/src/main/java/org/opengrok/indexer/web/Util.java?r=81b586e6#1319